internal/language_features/includes.go

```diff
@@ -115,8 +123,12 @@ func (f *IncludesFeature) getDefinitionsHover(includeName string) protocol.Hover
 			node := doc.Ast.RootNode().NamedDescendantForPointRange(referenceRange.StartPoint, referenceRange.EndPoint)
 			if node != nil {
 				result = append(result, protocol.HoverResultWithFile{
-					Value: node.Content([]byte(doc.Content)),
-					URI:   doc.URI,
+					Value: fmt.Sprintf(
+						`%shelm
+%s
```

Comment:
nit: wouldn't it be better to use double quotes and `\n`?

Reply:
I'm not sure how `\n` will behave on windows, I guess I will write a test for it.